build.gradle

```diff
@@ -9,16 +9,16 @@ buildscript {
     }
     dependencies {
         classpath "org.jlleitschuh.gradle:ktlint-gradle:10.3.0"
-        classpath 'com.google.dagger:hilt-android-gradle-plugin:2.44'
+        classpath 'com.google.dagger:hilt-android-gradle-plugin:2.44.2'
         classpath 'androidx.navigation:navigation-safe-args-gradle-plugin:2.5.3'
         classpath 'com.google.android.gms:oss-licenses-plugin:0.10.4'
     }
 }
 
 plugins {
-    id 'com.android.application' version '7.2.1' apply false
-    id 'com.android.library' version '7.2.1' apply false
-    id 'org.jetbrains.kotlin.android' version '1.6.10' apply false
+    id 'com.android.application' version '7.4.0' apply false
```

Comment:
Such things such not be mixed with code changes / adding new code. It is always better to extract adding/changing/bumping dependencies to another commit.

Reply:
Fixed here #138 :)